### DISCUSSION TITLE
Can we add a new feature that mark customize tag for each file, and search by tags?

### DISCUSSION THREAD

**Discussion by yyz17** [AlistGo/alist#7113] [Ideas]
Means users can maintain a tag library, mark the files and use the tags for search file, its cool, right? 
意思是用户可以维护一个标签库，并使用这些标签标记文件用来搜索，这个功能看上去不错。

**Comment by cidiancontrol** [comment]
赞同,我也很想要tag功能,之前想用nextcloud,可是性能太差,太笨重,很多功能用不上,alist好用多了可惜不能打标签
